### ISSUE TITLE
Escape key not working 

### ISSUE THREAD

**Issue by adamchaboryk** [joomla-projects/joomla-a11y-checker#63]
https://joomla-projects.github.io/joomla-a11y-checker/pages/pass.html

To test: Open accessibility checker, and press escape key. Throws console log error regarding `tippy.hideAll` function... I don't know why, your code looks identical to mine.

<img width="720" alt="Screenshot of console log showing error with tippy.hideAll function" src="https://user-images.githubusercontent.com/26335725/146273454-f7941638-d435-4fa2-a526-aa2e058b043f.png">

Also while you're within that keydown function feel free to include this extra snippet below. I'm adding this to my fork. Keyboard shortcut `Alt + A` to toggle checker on and off. 

````javascript
//Alt + A to enable accessibility checker.
if (evt.altKey && evt.code == "KeyA") {
					const sa11yToggle = document.getElementById("sa11y-toggle");
					sa11yToggle.click();
					sa11yToggle.focus();
					evt.preventDefault();
				}				
````

**Comment by brianteeman** [comment]
@fedik can you take a look please at the bug

**Comment by adamchaboryk** [comment]
Sorry, one more thing while we are on the topic of shortcuts. Optional enhancement if you'd like to keep consistency between libraries, I'm also binding the S key on the SkipToIssueToggle() function in addition to Alt + >. 

Rationale: S key is close to A key. S for "Skip", and easy one-handed keyboard control.

So...
Alt + A (Accessibility)
Alt + S or Alt + > (Skip to issue)

````javascript
//Jump to issue using keyboard shortcut.
document.addEventListener('keyup', (e) => {
				if (findSa11yBtn && (e.altKey && e.code == "Period" || e.code == "KeyS")) {
					skipToIssueToggle();
					e.preventDefault();
				}
			});			
````

**Comment by brianteeman** [comment]
We're going to be adding a shortcut key manager to joomla at some point to both manage existing keys and add new ones

**Comment by Fedik** [comment]
It happen after this #51 
The `hideAll` should be imported additionally https://atomiks.github.io/tippyjs/v5/methods/#hideall

But I suspect `tippy.hideAll();` call can be removed, because we have `this.resetAll()` the same as for "jooa11y-toggle".
I need to check more.

**Comment by Fedik** [comment]
That should fix it #64

**Comment by brianteeman** [comment]
Closed as these will all be in the next merge
